recipes/whatlies/meta.yaml

```
@@ -0,0 +1,116 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/whatlies-{{ version }}.tar.gz
+  sha256: 0b97c2b5b0bbc5c4044278ea8b2d3222cbb125e1fbf7bd854bb7f09b557e13d7
+
+build:
+  number: 0
+  noarch: python
+requirements:
+# ref: https://gitter.im/conda-forge/conda-forge.github.io?at=6195afbdb5ba9e5a11c02791
+  host:
+    - python >=3.6,<3.8
+  run:
+    - python >=3.6,<3.8
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
+      run_exports:
+        - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<3.8
+      run:
+        - python >=3.6,<3.8
+        # base_packages
+        - altair >=4.0.1
+        - bpemb >=0.3.0
+        - gensim ~=3.8.3
+        - matplotlib-base >=3.2.0
+        - scikit-learn >=0.24.0
+      run_constrained:
+        # umap_packages
+        - umap-learn >=0.4.0
+        # rasa_packages
+        - rasa >=2.7.0
+        # fasttext_packages
+        - fasttext ~=0.9.1
+        # spacy_packages
+        - spacy >=3.0.1
+        - spacy-lookups-data >=0.3.2
+        # s2v_packages (+= spacy_packages)
+        - sense2vec >=1.0.2
+        # tf_packages
+        - tensorflow >=2.3.0,<2.5
+        - tensorflow-text >=2.3.0
+        - tensorflow-hub >=0.8.0
+        # transformers_dep
+        - transformers >=4.3.0
+        # sentence_tfm_dep
+        - sentence-transformers >=0.3.8
+    test:
+      imports:
+        - whatlies
+      commands:
+        - pip check
+      requires:
+        - pip
+        # test_packages
+        #- torch >=1.6.0
+        #- torch
+        # - flake8 >=3.6.0
+        # - pytest >=4.0.2
+        # - black >=19.3b0
+        # - pytest-cov >=2.6.1
+        # - nbval >=0.9.5
+        # - pre-commit >=2.2.0
+
+  - name: {{ name|lower }}-umap
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
+    requirements:
+      host:
+        - pip
+        - python >=3.6,<3.8
+      run:
+        - python >=3.6,<3.8
+        # match major/minor version, but allow noarch
+        - {{ pin_subpackage('whatlies', max_pin="x.x") }}
+        - umap-learn >=0.4.0
+    test:
+      imports:
+        - whatlies
+        - umap
+      commands:
+        - pip check
+      requires:
+        - pip
+
+about:
+  home: https://rasahq.github.io/whatlies/
+  summary: Tools to help uncover `whatlies` in word embeddings.
+  license: Apache-2.0
+  license_file: LICENSE
+  description: |
+    This small library offers tools to make visualisation 
+    easier of both word embeddings as well as operations 
+    on them.
+  doc_url: https://rasahq.github.io/whatlies/
+  dev_url: https://github.com/RasaHQ/whatlies/
+
+extra:
+  recipe-maintainers:
+    - sugatoray
+    #- koaning
```

```suggestion
    #- koaning
  feedstock-name: {{ name|lower }}
```

This means that the name of the feedstock repo will be the name of the main package.